librubyfmt/build.rs

```diff
@@ -20,6 +20,13 @@ fn main() -> Output {
 
     let path = std::env::current_dir()?;
     let ruby_checkout_path = path.join("ruby_checkout");
+
+    if !ruby_checkout_path.join(".git").exists() {
```

Comment:
Do we actually need this conditional? Doesn't git already handle this?

Reply:
I copied and pasted this but probably not. In the worst case it runs the command when it doesn't need to and your build fails because github is down, but that seems like something this doesn't need to guard against.

Reply:
My gut says crates.io is more likely to be down that GitHub except I ran the ops for crates for years and by the end nearly every time I got paged it was because the index couldn't update because GH was down >_<

Reply:
haha

Reply:
It feels like checking an env variable to allow overriding the source location is something we might want to support eventually, and that would handle the case where someone wants to build with no live internet connection.

Definitely not something necessary in the short term.